link-paths.py

```diff
@@ -45,6 +45,24 @@
 template_linkname_model = outdir_model.resolve().as_posix()
 
 
+def sin_delta(altaz: AltAz):
+    """Delta is the angle between pointing and magnetic field."""
+    b_inc = u.Quantity(-37.36, u.deg)
+    b_dec = u.Quantity(-4.84, u.deg)
+
+    delta = angular_separation(
+        lon1=altaz.az,
+        lat1=altaz.alt,
+        lon2=b_dec,
+        lat2=b_inc,
+    )
+    return np.sin(delta.to_value(u.rad))
```

Comment:
I am pretty sure numpy can handle quantities in sin/cos.

Reply:
That's what I thought, but this way it is definitely correct :sweat_smile: